### ISSUE TITLE
Introduction

### ISSUE THREAD

**Issue by balacij** [joedf/CAS741_w23#10]
The introductory paragraph of your introduction is a bit wordy, at least the first sentence. Is there any chance you could rephrase some of it or add some pictures to quickly demonstrate what you mean to the reader?

Do you think you could expand your introduction a bit more?

**Comment by balacij** [comment]
I was going to write a separate ticket, but I think it's best I added a comment here:

Naively (e.g., from my POV :smile:), it's obvious why your case study is important, but I'm interested in hearing your thoughts. Do you think you can add a bit more of the rationale to your introduction about the need for your case study? e.g., what can happen when instruments are improperly tuned, how difficult is it to properly tune your instruments, is there any standards scientists need to follow when handling their instruments to create good research/etc? How can your program be used by ... scientists (?)... to improve their work?

**Comment by joedf** [comment]
@balacij Thanks for the thorough review! You raise some good points. I'll rework the intro as suggested.

I think partly why I was a bit more vague here is because I make the case for it in greater detail in the [ProblemStatement.pdf](https://github.com/joedf/CAS741_w23/blob/main/docs/ProblemStatementAndGoals/ProblemStatement.pdf)... but maybe I should do that a bit in the SRS too? I dunno? 🤷‍♂️

**Comment by smiths** [comment]
@joedf, you can't really assume that your reader has read your problem statement.  If there is some material in the problem statement that would help the SRS, you could copy it into the SRS.  If you would rather not do that, you could have an explicit pointer to the problem statement for motivation information.

**Comment by balacij** [comment]
Thank you @joedf, it was a great read, so time flew by :smile: 

@smiths' suggestion makes sense to me. Admittedly, I did not think to read your problem statement PDF.

**Comment by joedf** [comment]
@smiths thanks! That definitely clears it up for me.

Okay, I'll see what I can do.
